packages/jaeger-ui/src/components/SearchTracePage/SearchResults/ResultItem.tsx

```diff
@@ -19,6 +19,8 @@ import { Link } from 'react-router-dom';
 import { sortBy } from 'lodash';
 import moment from 'moment';
 
+import IoAlert from 'react-icons/lib/io/alert';
```

Comment:
I am not seeing this used in the other parts of the UI, how do we render (!) in the trace view?

Reply:
I'm using the same one as in line 152 of `SpanBarRow.tsx` to keep things consistent.
```
{showErrorIcon && <IoAlert className="SpanBarRow--errorIcon" />}
```